sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/cosmos/CosmosHealthIndicator.java

```diff
@@ -53,30 +52,25 @@ protected void doHealthCheck(Builder builder) {
             return;
         }
 
-        try {
-            CosmosDatabaseResponse response = this.cosmosAsyncClient.getDatabase(database)
-                                                                    .read()
-                                                                    .block(timeout);
+        CosmosDatabaseResponse response = this.cosmosAsyncClient.getDatabase(database)
+                                                                .read()
+                                                                .block(timeout);
 
-            if (response != null) {
-                LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
-                    response.getRequestCharge(), endpoint, database);
-                builder.up()
-                       .withDetail("RUs", response.getRequestCharge())
-                       .withDetail("CosmosUri", endpoint)
-                       .withDetail("database", database);
-            } else {
-                builder.down();
-            }
-        } catch (NotFoundException e) {
-            builder.status(Status.UNKNOWN).withDetail("Database not found",
-                "The option of `spring.cloud.azure.cosmos.database` is not configured correctly!");
+        if (response != null) {
+            LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
+                response.getRequestCharge(), endpoint, database);
+            builder.up()
+                   .withDetail("RUs", response.getRequestCharge())
+                   .withDetail("CosmosUri", endpoint)
+                   .withDetail("Database", database);
```

Comment:
Let's keep it as a `database`, I am not sure there will someone who uses this detailed `database` in their monitoring system, so this would be like a public API.